linkedevents/settings.py

```diff
@@ -60,6 +60,15 @@ def get_git_revision_hash() -> str:
     ),
     ENABLE_EXTERNAL_USER_EVENTS=(bool, True),
     ENABLE_REGISTRATION_ENDPOINTS=(bool, False),
+    ESPOO_API_URL=(str, "https://api.espoo.fi/events/"),
+    ESPOO_API_QUERY_PARAMS=(
```

Comment:
As these are only used with event resource queries, I would rather name this `ESPOO_API_EVENT_QUERY_PARAMS` (I actually first thought these were used in every query when I saw the setting 🙂)